eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/ManagementInputLibrary.cs

```diff
@@ -171,6 +172,11 @@ private IReadOnlyDictionary<InputServiceMethod, InputClient> ConstructMethodClie
             return map;
         }
 
+        private ArmProviderSchema BuildArmProviderSchema()
+        {
+            return new ArmProviderSchema(ResourceMetadatas, NonResourceMethods);
```

Comment:
now in this class, let us update the implementation of `ResourceMetadatas` and `NonResourceMethods` to make them return `ArmProviderSchema.ResourceMetadatas` and `ArmProviderSchema.NonResourceMethods`.